Tevling/Pages/Login.razor.cs

```diff
@@ -2,11 +2,10 @@ namespace Tevling.Pages;
 
 public partial class Login : ComponentBase
 {
-    [Inject]
-    StravaConfig StravaConfig { get; set; } = null!;
+    [Inject] private StravaConfig StravaConfig { get; set; } = null!;
 
     [SupplyParameterFromQuery(Name = "returnUrl")]
-    private string ReturnUrl { get; set; } = string.Empty;
```

Comment:
We need the setter here. I got an error message about it when testing your branch.